app/models/patient.rb

```diff
@@ -81,6 +82,7 @@ class Patient
   field :pledge_sent, type: Boolean
   field :resolved_without_fund, type: Boolean
   field :pledge_generated_at, type: DateTime
+  field :pledge_generated_by, type: String
```

Comment:
After making the `belongs_to` change, we should be able to ditch this line (since the `belongs_to` will define this field instead), so go ahead and strip this, if that's okay.